### ISSUE TITLE
projects Update and Squirrel.Test in change afb6817 have build errors.

### ISSUE THREAD

**Issue by potentZilla** [Squirrel/Squirrel.Windows#364]
I am unable to build the full solution and what it appears is the edits around the ILMerge strongly named binary with the projects Update and Squirrel.Tests might be incomplete.

The Update and Squirrel.Tests projects have had the Nuget.Core assembly reference removed from the .csproj but the projects still contain code that uses classes and interfaces from Nuget.Core.  In addition packages.config still has the package listed so when looking at the nuget package manager it appears Nuget.Core is already added to the projects.

After deleting the entries in the packages.config and then adding Nuget.Core back to the projects the solution builds.  Although due to the comments on the commits I am not sure if this is desired or if there is another method to get the source to build.

**Comment by potentZilla** [comment]
Any input on this situation?  It seems pretty easy to reproduce.  I would like to contribute some unit tests to validate a situation but I would also like to be able to confirm the status of the code.

I don't understand how this would be only an issue I would have unless there are build environment differences between what I have and everyone else who builds the latest change set.

**Comment by potentZilla** [comment]
I am on change set afb6817a8d490de4aee8e2e4119b1aa0a8bd99e0.  After I have corrected the build errors on the two projects.   I have the following unit test status.

13 Tests Failed
3 Tests Skipped
81 Tests Passed.

Would this be an expected unit test summary for the change set?

**Comment by potentZilla** [comment]
I have found that the project's msbuild contains conditions based upon the build configuration.  The build will not work using "Debug" build configuration.  The Release build configuration worked.  I am avoiding the CIBuild configuration because after attempting it I have had to reinstall Visual Studio.  Although antidotal evidence.  This is obviously a build configuration reserved for a specific environment.
